test/tools/libtesteth/ImportTest.cpp

```diff
@@ -289,11 +296,18 @@ json_spirit::mObject& ImportTest::makeAllFieldsHex(json_spirit::mObject& _o, boo
 			str = value.get_str();
 		else if (value.type() == json_spirit::array_type)
```

Comment:
it does. but to remove '//' prefixed fields. 
make all fields hex takes an object to convert. an object like 
"key" : "value" 

this case describes a situation 
"key" : [
     "value" 
     "value"
]